### PR TITLE
[SEM-35]: Implementation of Buttons in the Sidebar Menu for Managing Articles and Authors

### PR DESCRIPTION
**Pull request by Then-Crimson**
### Description
Implemented "Manage Articles" and "Manage Authors" buttons in the sidebar to improve navigation.

### Acceptance Criteria
- Buttons integrated as per design.
- Icons and labels correctly aligned and sized.

### Storybook Implementation
- Entries created for Normal and Hover states.
- Controls added to test different states.

### TDD
- Unit tests for navigation and state changes.
- Coverage for color changes and alignment.

### Notes
- CSS used for styling and compatibility.

### Attachments
image
image

### CODE REVIEW

File: src/ui/components/atoms/side-menu-button/SideMenuButton.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  };
+
+  const selectedIcon = icon ?? (children === 'Article Management' ? Icon1 : Icon2);
+
+  return (
+    <button
+      data-testid="side-menu-button-styles"
+      className={`${styles['side-menu-button']} ${styles[color ?? '']} ${
+        variant ? styles[variant] : ''
+      } ${isActive ? styles.active : ''}`}
```
Comment (Ditmar) [inline src/ui/components/atoms/side-menu-button/SideMenuButton.tsx:24]: this logic will be difficult to maintain in the future. please use this approach https://github.com/Ditmar/backoffice-openscience/blob/master/src/ui/components/demo/calculator/atoms/display.tsx#L7-L10

**Review by Ditmar — CHANGES_REQUESTED** [review]
good work, please read the comments

**Comment by Then-Crimson** [comment]
https://ditmar.github.io/backoffice-openscience/branch-feature/SEM-035/?path=/story/ui-components-atoms-side-menu-button--article 
Storybook--

**Comment by Then-Crimson** [comment]
https://ditmar.github.io/backoffice-openscience/branch-SEM-035/?path=/story/ui-components-atoms-side-menu-button--article
for storybook

**Comment by Then-Crimson** [comment]
https://ditmar.github.io/backoffice-openscience/branch-SEM-035/?path=/story/ui-components-atoms-side-menu-button--article R3.0

**Comment by Then-Crimson** [comment]
https://ditmar.github.io/backoffice-openscience/branch-SEM-035/?path=/story/ui-components-atoms-side-menu-button--article
R4.0

**Review by Ditmar — APPROVED** [review]

**Pull request merged**